server/src/main/java/org/opensearch/plugins/PluginInfo.java

```diff
@@ -491,7 +491,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         {
             builder.field("name", name);
             builder.field("version", version);
-            builder.field("opensearch_version", opensearchVersionRanges);
+            builder.field("opensearch_version", getOpenSearchVersionRangesString());
```

Comment:
Thanks for the super quick fix. Should we do an additional pass at all places of usage to confirm they are bwc?

Reply:
All callers that require a serialized form of PluginInfo will use one of the following:
1. writeTo method - already updated to be bwc
2. toXContent method - updated with this PR
3. toString method - already updated to be bwc